pkg/defaults/defaults.go

```diff
@@ -48,7 +48,7 @@ const (
 	DefaultRegistryPort = 5000
 
 	//tags, versions, repos
-	DefaultEVETag               = "5.21.1" //DefaultEVETag tag for EVE image
+	DefaultEVETag               = "0.0.0-master-a8c5dddf" //DefaultEVETag tag for EVE image
```

Comment:
What was changed in Eve so we can't reboot in 5.21.1 
I'd like to have a rule of thumb that we rely on tagged versions?

Reply:
It is here: https://github.com/lf-edge/eve/pull/1892

Actually, we can wait for the next tagged version

Reply:
I see ... good option

Reply:
So, now it has tag